UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -9,34 +9,38 @@
 public class GameManager : MonoBehaviour
 {
 	public static GameManager Instance;
-	public float RoundTime = 480f;
-	public bool counting;
+	public float RoundTime = 600f;
```

Comment:
You're using spaces instead of tabs, please use tab delimiters for consistency with our codebase